examples/supermassive-todomvcs-apollo-style/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "supermassive-todomvc",
+  "main": "index.js",
+  "private": true,
+  "version": "1.0.0",
+  "scripts": {
+    "start": "graphql-codegen && yarn serve --progress --color --mode=development",
+    "build": "webpack --mode production --config webpack.config.ts",
+    "serve": "webpack serve --config webpack.config.ts",
+    "extract-schema": "supermassive extract-schema ./src/schema/typeDefs.graphql",
+    "codegen": "graphql-codegen"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.6.0",
+    "@babel/runtime": "^7.12.0",
+    "@graphitation/apollo-react-relay-duct-tape": "^0.7.0",
+    "@graphitation/graphql-codegen-supermassive-relay-ir-plugin": "link:../packages/graphql-codegen-supermassive-relay-ir-plugin",
```

Comment:
@freiksenet I had to resort to this, as I couldn't get `yarn link` to work for the life of me.